src/SDiagonal.jl

```diff
@@ -82,10 +82,16 @@ function logdet(D::SDiagonal{N,T}) where {N,T<:Complex} #Make sure branch cut is
 end
 
 eye(::Type{SDiagonal{N,T}}) where {N,T} = SDiagonal(ones(SVector{N,T}))
+one(::Type{SDiagonal{N,T}}) where {N,T} = SDiagonal(ones(SVector{N,T}))
+one(::SDiagonal{N,T}) where {N,T} = SDiagonal(ones(SVector{N,T}))
```

Comment:
If I read Coverall correctly, `one(::SDiagonal{N,T})` is not needed for recursive cholesky. Should I leave this?